.circleci/local_publish_helpers.sh

```diff
@@ -250,8 +250,8 @@ function setAwsAccountCredentials {
     export AWS_ACCESS_KEY_ID_ORIG=$AWS_ACCESS_KEY_ID
     export AWS_SECRET_ACCESS_KEY_ORIG=$AWS_SECRET_ACCESS_KEY
     export AWS_SESSION_TOKEN_ORIG=$AWS_SESSION_TOKEN
-    # introduce a delay of up to 5 minutes to allow for more even spread aws list-accounts calls due to throttling
-    sleep $[ ( $RANDOM % 300 )  + 1 ]s
```

Comment:
i've reduced this a lot because the number of executors has come down drastically, and it appears to be working fine

from ~490 to ~280